processor/processor.go

```diff
@@ -949,6 +949,9 @@ func (proc *HandleT) getFailedEventJobs(response transformer.ResponseT, commonMe
 	failedCountMap := make(map[string]int64)
 	var failedEventsToStore []*jobsdb.JobT
 	for _, failedEvent := range response.FailedEvents {
+		if failedEvent.StatusCode == transformer.DROP_STATUS_CODE {
+			continue
```

Comment:
Would it make sense to bump some metric here to keep track of these?

Reply:
It would.